libnmstate/nm/plugin.py

```diff
@@ -43,6 +46,7 @@
 
 class NetworkManagerPlugin:
     def __init__(self):
+        self.check_version_mismatch()
         self._ctx = NmContext()
```

Comment:
This is wrong. It should be:
```
self._ctx = NmContext()
self._check_version_mismatch()
```
This way the context is initialized and you are going to be able to do `self._ctx.client()` in `check_version_mismatch()`. Please, see the comments below.

Reply:
Done, thank you!